src/core/src/service_interfaces/TelemetryWriter.py

```diff
@@ -28,6 +28,8 @@
 class TelemetryWriter(object):
     """Class for writing telemetry data to data transports"""
 
+    machine_info = ""
+
```

Comment:
Instantiate this under __init__ with self.machine_info

Reply:
We don't need to initialize it. We can directly set it in set_machine_config_info() function which is called by init function